src/python/grpcio/grpc/__init__.py

```diff
@@ -1360,6 +1360,19 @@ def method_handlers_generic_handler(service, method_handlers):
     return _utilities.DictionaryGenericHandler(service, method_handlers)
 
 
+def ssl_session_cache_lru(capacity):
+    """Creates an SSLSessionCacheLRU
+
+    Args:
+      capacity: Size of the cache
+
+    Returns:
+      An SSLSessionCacheLRU. This is typically passed to a grpc.Channel when a
```

Comment:
Where is SSLSessionCacheLRU defined in the public API of gRPC Python? How does one pass it to a `grpc.Channel`?

Reply:
Hmm, it's not in the public API, so I suppose the comment here isn't useful. Clients pass the cache returned by this call as a `grpc.ssl_session_cache` channel option  like so: https://github.com/grpc/grpc/pull/14879/commits/8ac9086f71266730867dbe82754c1484de733abd#diff-dd4ea0e25f29357373b0e0911894153aR173

Clients are not expected to directly create an `SSLSessionCacheLRU` except through this call. Should we add it to the API anyway, or omit the reference to it in the comment here?

Reply:
`ssl_session_cache_lru` now returns an `SSLSessionCache` type. Added `SSLSessionCache` to the Python API, and made the comments a bit clearer.